From 0e2a3a00851da750989ac5cde44ba69ca18b7332 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Tue, 26 Jan 2021 03:01:10 +0100
Subject: [PATCH] Fix gitless error

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 gitless/cli/gl.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gitless/cli/gl.py b/gitless/cli/gl.py
index 3723ac3..67bed8a 100644
--- a/gitless/cli/gl.py
+++ b/gitless/cli/gl.py
@@ -39,7 +39,7 @@ try:
   try:
     pprint.DISABLE_COLOR = not repo.config.get_bool('color.ui')
   except pygit2.GitError:
-    prrint.DISABLE_COLOR = (
+    pprint.DISABLE_COLOR = (
         repo.config['color.ui'] in ['no', 'never'])
 except (core.NotInRepoError, KeyError):
   pass
-- 
2.30.0.155.g66e871b664

