From 98e3a92715463a6045b37a417b2a6d9de04f5d27 Mon Sep 17 00:00:00 2001
From: Piotr Gorski <lucjan.lucjanov@gmail.com>
Date: Mon, 7 Dec 2020 23:48:05 +0100
Subject: [PATCH] meson: update version and libalpm_version

Signed-off-by: Piotr Gorski <lucjan.lucjanov@gmail.com>
---
 meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/meson.build b/meson.build
index 264c6501..a1cf6f04 100644
--- a/meson.build
+++ b/meson.build
@@ -1,6 +1,6 @@
 project('pacman',
         'c',
-        version : '5.2.1',
+        version : '6.0.0',
         license : 'GPLv2+',
         default_options : [
           'c_std=gnu99',
@@ -10,7 +10,7 @@ project('pacman',
         ],
         meson_version : '>= 0.51')
 
-libalpm_version = '12.0.1'
+libalpm_version = '12.0.2'
 
 cc = meson.get_compiler('c')
 
-- 
2.29.2.435.g72ffeb997e

